assets/scripts/oppia-player-0.0.3.js

```diff
@@ -0,0 +1,363 @@
+    // changed during the for loop.
+    var oppiaElements = [];
+    for (var i = 0; i < oppiaElementsFromDOM.length; i++) {
+      oppiaElements.push(oppiaElementsFromDOM[i]);
+    }
+
+    for (var i = 0; i < oppiaElements.length; i++) {
+      var oppiaElement = new OppiaEmbed(oppiaElements[i]);
+      oppiaElement.init();
+    }
+  };
+
+  window.OPPIA_PLAYER = {
+    /**
+     * Called when the height of the embedded iframe is changed.
+     * @param {object} iframeNode The iframe node that is the source of the
+     *     postMessage call.
+     * @param {int} newHeight The new height of the embedded iframe.
+     * @param {boolean} doScroll Scroll down to show the bottom of iframe after
+     *     changing the height.
+     */
+    onHeightChange: function(iframeNode, newHeight, doScroll) {
+      _debug(
+        'onHeightChange event triggered on ' + iframeNode + ' for ' +
+        newHeight);
+
+      // This is set to 'auto' when the exploration is reset. If this is not
+      // removed, the iframe height will not change even if iframeNode.height
+      // is set.
+      iframeNode.style.height = '';
+
+      if (iframeNode.getAttribute('fixedheight') === 'false') {
+        iframeNode.height = newHeight + 'px';
+      }
+      if (doScroll) {
+        iframeNode.scrollIntoView(false);
+      }
+
+      window.OPPIA_PLAYER.onHeightChangePostHook(iframeNode, newHeight);
+    },
+    onExplorationLoaded: function(iframeNode, explorationName) {
+      setTimeout(function() {
+        // Show the oppia contents after making sure the rendering happened.
+        iframeNode.style.position = 'inherit';
+        iframeNode.style.visibility = 'inherit';
+        iframeNode.style.top = 'inherit';
+      }, 0);
+      window.OPPIA_PLAYER.onExplorationLoadedPostHook(
+        iframeNode, explorationName);
+    },
+    onStateTransition: function(
+        iframeNode, oldStateName, jsonAnswer, newStateName) {
+      window.OPPIA_PLAYER.onStateTransitionPostHook(
+        iframeNode, oldStateName, jsonAnswer, newStateName);
+    },
+    onExplorationReset: function(iframeNode, stateName) {
+      window.OPPIA_PLAYER.onExplorationResetPostHook(iframeNode, stateName);
+    },
+    onExplorationCompleted: function(iframeNode) {
+      window.OPPIA_PLAYER.onExplorationCompletedPostHook(iframeNode);
+    }
+  };
+}(window, document));
+
+// FIXME: The contents of all functions below this line can be overwritten.
+
+/**
+ * Called after the height of the embedded iframe is changed and the iframe has
+ * been appropriately resized.
+ * @param {object} iframeNode The iframe node that is the source of the
+ *     postMessage call.
+ * @param {int} newHeight The new height of the embedded iframe.
+ */
+window.OPPIA_PLAYER.onHeightChangePostHook = function(iframeNode, newHeight) {
+
+};
+
+/**
+ * Called when the exploration is loaded.
+ * @param {object} iframeNode The iframe node that is the source of the
+ *     postMessage call.
```

Comment:
Missing docstring for explorationName (don't forget to re-minify after you fix it)

Reply:
Done! (and re-minified it!)